src/index.ts

```diff
@@ -82,6 +85,126 @@ export const AWS: SDK = new Proxy({} as any, {
           };
         }
 
+        function createSnsHandler(methodName: string) {
+          return async (input: any) => {
+            const url = new URL(endpoint);
+            for (const [k, v] of Object.entries(input)) {
+              if (k !== "PublishBatchRequestEntries") {
+                url.searchParams.set(k, v as any);
+              } else {
+                for (const [i, entry] of (v as any[]).entries()) {
+                  for (const [k, v] of Object.entries(entry)) {
+                    url.searchParams.set(
+                      `PublishBatchRequestEntries.${k}.${i}`,
+                      // `PublishBatchRequestEntries.${k}.${i + 1}`,
+                      // `PublishBatchRequestEntry.${i + 1}.${k}`,
+                      // `PublishBatchRequestEntries.${i + 1}.${k}`,
```

Comment:
Are these valid? Are we supporting all possible cases or a subset?

Reply:
Sorry, the PR was meant to be a draft, there's many things I need to clean up first

Reply:
Finally found the format. It's literally `PublishBatchRequestEntries.member.${i + 1}.${k}`.